02-cpu-configure/pcie-remove.c

```diff
@@ -0,0 +1,115 @@
+/* BPM PCIe Utils
+ * Copyright (C) 2024 CNPEM
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <fcntl.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+
+/*
+ * Slot whitelist to sanitize user input
+ */
+const char* const slot_whitelist[] = {
+    "1",
+    "2",
+    "2-1",
+    "3",
+    "4",
+    "5",
+    "6",
+    "7",
+    "8",
+    "9",
+    "10",
+    "11",
+    "12",
+};
+
+/*
+ * Check if the user provided slot string is valid
+ */
+int sanitize_slot(const char* slot) {
+    const ssize_t items = sizeof(slot_whitelist) / sizeof(char*);
+    int ret = -1;
+
+    for (ssize_t i = 0; i < items; i++) {
```

Comment:
For such values you'd use `size_t` rather than `ssize_t`, since you know these values can only be unsigned

Reply:
done.